scripts/mkcloud

```diff
@@ -134,6 +136,7 @@ fi
 pidfile=mkcloud.pid
 
 trap 'error_exit $? "error caught by trap"' TERM
+exec 3<&0
```

Comment:
`crowbar_dev_server` is also interactive btw. 

I just wanted to wrap all into one central tool to make it easier and more obvious to use. Using extra scripts feels a bit like doing it the old way by calling everything manually.
To me it actually doesn't matter but someone who will actually use it in the end should give an opinion as well.